### DISCUSSION TITLE
Convert boolean to string c#

### DISCUSSION THREAD

**Discussion by Gabs1993** [serenity-is/Serenity#6371] [Help]
Hi guys

I'm trying to convert a boolean to string in c# and exporting an excel file, this file coming true, but I want to transform to yes/no. I've tried every way, got it on multiple platforms, so far, could anyone help?

my code is like this

public boolean? ForeignQuar
{

        get => fields.foreign[this];
        set => fields.foreign[this] = value;
    }

**Comment by ra-bedo11** [comment]
public string? Gender
        {
            get => (string?) fields.Gender[this];
            set => fields.Gender[this] = (bool?)value;
        }

**Reply by minhhungit** [reply]
Can I ask why you have to convert string to bool, almost case you won't need to do it, I think just use boolean is the best practics 
You can use "editor" or "formatter" to custom UI

**Reply by Gabs1993** [reply]
I thought about converting to string initially because I need to export with "yes/no" values ​​and currently it just imports true or false. About the editor and formatter, is there any that do this?

**Reply by minhhungit** [reply]
You can create a new property for the custom value and use Expression, for example
```csharp
[Expression("CASE WHEN Gender = 1 THEN 'Yes' ELSE 'No' END")]
public string? CustomField
{
    get => fields.CustomField[this];
    set => fields.CustomField[this] =value;
}
```

**Reply by stixoffire** [reply]
You can also do this on an Enumeration, Description Yes / No.
